### PR TITLE
Multiple dev reward addresses

### PR DESCRIPTION
**Pull request by antstalepresh**

### CODE REVIEW

File: x/pool-incentives/keeper/distr_test.go [hunk 1]
```diff
@@ -46,7 +52,12 @@ func (suite *KeeperTestSuite) TestAllocateAsset() {
 	keeper := suite.app.PoolIncentivesKeeper
 	mintKeeper := suite.app.MintKeeper
 	params := suite.app.MintKeeper.GetParams(suite.ctx)
-	params.DeveloperRewardsReceiver = sdk.AccAddress([]byte("addr1---------------")).String()
+	params.WeightedDeveloperRewardsReceivers = []minttypes.WeightedAddress{
```
Comment (sunnya97) [inline x/pool-incentives/keeper/distr_test.go:55]: Can we some tests with multiple addresses

File: x/mint/types/params.go [hunk 2]
```diff
@@ -219,19 +219,38 @@ func validateDistributionProportions(i interface{}) error {
...
-	if v == "" {
+	if len(v) == 0 {
 		return nil
 	}
 
-	_, err := sdk.AccAddressFromBech32(v)
-	return err
+	weightSum := sdk.NewDec(0)
+	for i, w := range v {
+		// TODO: should we allow portion to be "" to go to community pool?
```
Comment (sunnya97) [inline x/mint/types/params.go:235]: Yeah I suppose that's probably a good idea.

Comment (antstalepresh) [reply x/mint/types/params.go]: Added

File: x/pool-incentives/keeper/distr_test.go (hunk 1 above)
Comment (antstalepresh) [reply x/pool-incentives/keeper/distr_test.go:55]: Added

**Comment by ValarDragon** [comment]
Sorry, I reviewed this much earlier in the day, and just forgot to press the send review button before entering meetings. LGTM with my comments, will merge once tests pass

**Pull request merged**